sonar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "Building docker image"
+docker build -f ./config/sonar.dockerfile -t sonar ./config/
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonar:latest
```

Comment:
docker start/stop will done by wercker or any other CI we choose for this. So you need this to be done in separate wrapper shell file that wrap execution of shell file that do all configration of sonar and validation.
You can reuse "checkstyle/sonarqube-maven-git" image that we use at https://github.com/checkstyle/sonar-checkstyle/blob/master/wercker.yml.

CI docker config file will take care of "git clone" for checkstyle repo inside docker container.

Wrapper shell file(file that will be used only by humans) will run docker image and do clone of sources inside container and execution of validation shell file.

Validation shell file will be used by CI and Wrapper shell script in the same way.

You need to have a validation shell file that is executed inside a docker container and do:
- configure sonar server by configs from checkstyle repo
- execute validation against checkstyle sources 
- check result of validation.
- print some output 
- return exit code